src/typings/vscode.proposed.d.ts

```diff
@@ -0,0 +1,2104 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I don't think we need to add this file, running `yarn` should pull the latest typings from `master` automatically. Is that not working?

Reply:
You're right, removed!